Doc/c-api/object.rst

```diff
@@ -196,6 +196,17 @@ Object Protocol
       This function now includes a debug assertion to help ensure that it
       does not silently discard an active exception.
 
+
+.. c:function:: PyObject* _PyObject_FunctionStr(PyObject *func)
```

Comment:
It was added to functions that will be finalized in 3.9, but elsewhere, documenting private functions is not very helpful.

Reply:
Agree with @encukou.

Reply:
> documenting private functions is not very helpful

Intentionally not documenting useful functions is even less helpful. But since you insist, I will remove it.